scripts/setup.py

```diff
@@ -142,7 +142,10 @@ def download_and_install_node() -> None:
         if common.is_x64_architecture():
             if common.is_mac_os():
                 node_file_name = 'node-v%s-darwin-x64' % (common.NODE_VERSION)
-            elif common.is_linux_os():
+            # Oppia only suppports windows, mac and linux operating systems.
+            # Hence we use the 'no branch' flag since the else condition will
+            # never occur.
```

Comment:
In line with what we discussed on the M2.1 PR, maybe add an `else` condition that throws an exception and put `# pragma: no branch` on that instead?

Reply:
Done. 
We would have to use `pragma: no cover` flag since now the line containing the Exception will be uncovered too.